webapp/src/main/webapp/news/components/NewsApp.vue

```diff
@@ -287,7 +290,7 @@ export default {
           newsText: this.getNewsText(item.summary, item.body),
           illustrationURL: `${newsIllustration}?${newsIllustrationUpdatedTime}`,
           title: item.title,
-          creationDate: newsPublicationDate,
+          creationDate: item.publicationState !== 'draft' ? newsPublicationDate : newsUpdateDate,
```

Comment:
to verify if it is not always update date

Reply:
Verified!

Reply:
we can change this to updatedDate, it is more coherent IMHO